google/cloud/security/scanner/scanner.py

```diff
@@ -195,12 +190,8 @@ def _get_project_policies(timestamp):
         The project policies.
     """
     project_policies = {}
-    try:
-        dao = project_dao.ProjectDao()
-        project_policies = dao.get_project_policies('projects', timestamp)
-    except MySQLError as err:
```

Comment:
Hmmm, is this MySQLError being handled elsewhere now?  IIRC, I was working with this part of the code earlier, and this was the only place where the MySQLError would be caught when thrown by the dao.